app/ai/text/blueprint/api.py

```
@@ -0,0 +1,203 @@
+            ),
+            architect_model=request.architect_model,
+            enhancer_model=request.enhancer_model,
+            reviewer_model=request.reviewer_model,
+        )
+
+        return GenerateTemplateResponse(
+            status=result["status"],
+            message=result["message"],
+            result=result.get("result"),
+        )
+    except Exception as e:
```

Ruff (0.15.2)

[warning] Do not catch blind exception: `Exception`

(BLE001)